### ISSUE TITLE
Blank Autogenerated Card/Image Appearing for Article on Twitter

### ISSUE THREAD

**Issue by michael-tharrington** [thepracticaldev/dev.to#2907]
**Describe the bug**
The autogenerated social card that appears with articles on Twitter when they don't have a cover image is blank.

**To Reproduce**

1. Go to post an article - https://dev.to/new
2. Add the title "Postman, Ajax endpoints and XSRF tokens"
3. Write enough info to succesfully save the post, and set publish to true
4. Copy URL of post and visit https://cards-dev.twitter.com/validator to confirm blank image for card

**Expected behavior**
Autogenerated images should include titles of posts.

**Screenshots**

Here's how the article's image appears blank on Twitter - 

![](https://cl.ly/40e7dee99d11/download/Image%2525202019-05-20%252520at%2525205.25.59%252520PM.png)

Here's my recreation of it:

![](https://cl.ly/44dc8c2236ed/download/Image%2525202019-05-20%252520at%2525205.41.31%252520PM.png)

Here's an example of another article with a long title; the Twitter card was visible without problems:

![](https://cl.ly/d004053555f9/download/Image%2525202019-05-20%252520at%2525205.35.13%252520PM.png)

**Additional context**
I've replaced the Twitter card image for the article on Twitter with a free commons unsplash image so that it should appear normally.

**Comment by jessleenyc** [comment]
@mscoutermarsh do you have any thoughts on this?! @michael-tharrington was able to recreate this with the same title but it's not happening to any other titles.

**Comment by mscoutermarsh** [comment]
@jessleenyc weird! no ideas off top of my head. Going to look 👀

**Comment by mscoutermarsh** [comment]
@jessleenyc @michael-tharrington 

This should be fixed now. I'll keep an eye on it.

I looked through hundreds of images and found a couple where this happens 😄. I believe we are seeing "FOIT" aka "Flash of Invisible Text" that can happen when a google font loads slowly. I tweaked our google fonts script to better handle for this. So we shouldn't see this happening anymore.

@michael-tharrington thank you for the detailed bug report 🎉
